### PR TITLE
Alt list arrow update

### PR DESCRIPTION
**Pull request by theilman7**
Minor update, changing the arrow styling for alt-list. Switching from f054 to f0a9.

### CODE REVIEW

File: src/styles/partials/components/_alt-list.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@ $icon-arrow-width: 32px;
     &:after {
       color: $gray-light;
       @include default-transition;
-      @include icon-content($icon-arrow-right);
+      @include icon-content($icon-arrow-right-circle);
```
Comment (martypowell) [inline src/styles/partials/components/_alt-list.scss]: seems like you want to update the icon-arrow-right, instead of circle so it's update throughout?

Comment (martypowell) [reply src/styles/partials/components/_alt-list.scss]: or is this only targeting links?

Comment (theilman7) [reply src/styles/partials/components/_alt-list.scss]: @martypowell I don't think we want to do that, as icon-arrow-right variable is used for accordions currently. This just adds an updated icon to the alt-list component.

Comment (martypowell) [reply src/styles/partials/components/_alt-list.scss]: to make it easier on the developers, can we rename this osmething maybe like `$icon-arrow-link` or something along those lines, something that might make a little more sense?

Comment (theilman7) [reply src/styles/partials/components/_alt-list.scss]: Makes sense to me. Updated and committed.

Comment (theilman7) [reply src/styles/partials/components/_alt-list.scss]: @martypowell Ready for a second look when you have a moment.

**Review by martypowell — APPROVED** [review]

**Pull request merged**